app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,142 @@
+class ProjectsController < ApplicationController
+  load_and_authorize_resource
+  def index
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def new
+    respond_to do |format|
+      format.js
+    end
+  end
+
+  def create
+    compnay = Company.first
+    @project.name = params[:project][:name]
+    @project.project_category = params[:project][:project_category]
+    @project.company = compnay
+    respond_to do |format|
+      if @project.save
+        flash[:success] = 'Project Created Successfully'
```

Comment:
Use internationalization for all the text that is displayed to user. Verify everywhere.